advocacy_docs/security/securing-postgresql/301/index.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: Securing PostgreSQL 301
+navTitle: Security 301
+description: Your guide to Compliance, certifications, auditing and other higher-level issues.
+---
+
+As security requirements increase in complexity, it’s critical to move beyond basic and intermediate configurations. 
+Advanced security in PostgreSQL focuses on hardening systems to meet strict compliance standards, such as Security Technical Implementation Guides (STIGs), GDPR, PCI-DSS, HIPAA, and FISMA. 
+Use the following advanced strategies to secure PostgreSQL in high-stakes environments.
+
+## Security Technical Implementation Guides (STIGs)
+
```

Comment:
Shouldn't this give an indication as to where to find Stigs? And note there's Stigs for Postgresql and for EDB Postgres Advanced Server (at least)